src/CommunityToolkit.Maui.Core/Extensions/ColorConversionExtensions.shared.cs

```diff
@@ -7,6 +7,18 @@ namespace CommunityToolkit.Maui.Core.Extensions;
 /// </summary>
 public static class ColorConversionExtensions
 {
+	/// <summary>
+	/// Converts the value to percentage.
+	/// Uses <see cref="MidpointRounding.ToEven"/> and "0%" format string to emulate the behavior of "en-US" locale.
+	/// If only "0%" is used to get rid of the space between the value and the percent sign, 0.625f would become 63% instead of 62%.
+	/// </summary>
+	/// <param name="percentage">percentage</param>
+	/// <returns></returns>
+	static string ToPercentage(this float percentage)
```

Comment:
I would like to get some feedback on this method. I don't really like it, but I wanted to ensure the same rounding behavior as before my commit.

If the change in rounding behavior is ok. I would prefer to use only "0%", remove the method and update the failing tests.
```
failed CommunityToolkit.Maui.UnitTests.Converters.ColorToHslaStringConverterTests.ColorToRgbStringConverterValidInputTest(red: 0,25, green: 1, blue: 0,25, alpha: 1, expectedResult: "HSLA(120,100%,62%,1)")[en-US] (0ms)
  Assert.Equal() Failure: Values differ
  Expected: HSLA(120,100%,62%,1)
  Actual:   HSLA(120,100%,63%,1)
```